gemrb/core/GUI/GameControl.cpp

```diff
@@ -2079,6 +2075,15 @@ void GameControl::InitFormation(const Point &clickPoint)
 	SetCursor(core->Cursors[IE_CURSOR_USE]);
 }
 
+bool GameControl::TryDefaultTalk(Point p) const
+{
+	Actor *targetActor = core->GetGame()->GetCurrentArea()->GetActor(p,
+			target_types & ~(GA_NO_HIDDEN /*| GA_NO_ALLY | GA_NO_ENEMY*/ ));
+	if (targetActor)
+		targetActor->VerbalConstant(VB_SELECT, 4, DS_CIRCLE);
```

Comment:
The number of selection sounds differs per game, so that's why I suggested using the Actor method. If the originals ignored the settings in this case, just add a parameter to it.

Reply:
so i need to make a new actor method just to get rid of the magic constant?

Reply:
Is the existing one I suggested not fit for this purpose?